Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
Sorry but I still do not understand your point.

Yes, there's a number of ways in which `new Mock<IHasProperties>().Object` differs from `new Fixture().Create<Mock<IHasProperties>>().Object;`; one has its members setup and its properties populated, while the other one doesn't, for example. But that's the whole point of the `AutoConfiguredMoqCustomization`.